core/integration/container_test.go

```diff
@@ -2935,11 +2935,11 @@ func TestContainerInsecureRootCapabilites(t *testing.T) {
 		}).
 		Stdout(ctx)
 	require.NoError(t, err)
-	require.Contains(t, out, "CapInh:\t0000003fffffffff")
-	require.Contains(t, out, "CapPrm:\t0000003fffffffff")
-	require.Contains(t, out, "CapEff:\t0000003fffffffff")
-	require.Contains(t, out, "CapBnd:\t0000003fffffffff")
-	require.Contains(t, out, "CapAmb:\t0000003fffffffff")
+	require.Contains(t, out, "CapInh:\t0000001fffffffff")
+	require.Contains(t, out, "CapPrm:\t0000001fffffffff")
+	require.Contains(t, out, "CapEff:\t0000001fffffffff")
+	require.Contains(t, out, "CapBnd:\t0000001fffffffff")
+	require.Contains(t, out, "CapAmb:\t0000001fffffffff")
```

Comment:
I guess Buildkit drops some capabilities by default now?

Reply:
Yeah I'm not sure exactly why this changed, probably a transitive dependency on buildkit, but from `capsh --decode=0000001fffffffff` it looks like `cap_audit_read` is not present now.

That capability will "allow reading the audit log via a multicast netlink socket", which to my knowledge is very obscure, so I don't think it's worth caring about right now.

Reply:
Well now the test failed in CI because there it apparently is still `0000003fffffffff` ??? 

I don't know why this is different between my box and CI but I think I'll just update this test to decode the capabilities and assert that the major ones like cap_sys_admin are there.